More links updated to @uhermjakob's pages

austinjp opened the pull request:
I've removed a link to the 'list of frame arguments in PropBank (AMR version)' at https://amr.isi.edu/doc/propbank-amr-frames-arg-descr.txt since I'm not sure what to replace it with.

Perhaps https://propbank.github.io/ ?

austinjp committed:
More links update to @uhermjakob's pages

I've removed a link to the 'list of frame arguments in PropBank (AMR version)' at https://amr.isi.edu/doc/propbank-amr-frames-arg-descr.txt since I'm not sure what to replace it with.

Perhaps https://propbank.github.io/ ?

austinjp commented:
See other pull request #277 

Probably both relate to issue #276

flipz357 commented:
Don't wanna get in conflict with your PR by making another one --- maybe you can integrate these two links:

propbank-amr-frames-arg-descr.txt (don't need to remove):

https://github.com/flipz357/AMR-World/blob/main/data/dictionaries/propbank-amr-frames-arg-descr.txt

Little Prince, Bio AMR data resources: 

https://github.com/flipz357/AMR-World/tree/main/data/reference_amrs

austinjp committed:
Updated with links from @flipz357

As suggested in #278 here: https://github.com/amrisi/amr-guidelines/pull/278#issuecomment-2480601730

austinjp committed:
Added Juri's name :)

nschneid commented:
Would it be better to link to the pages on the UMR website? https://umr4nlp.github.io/web/amr_resources.html

flipz357 commented:
Didn't see this. Yes this can work too (though the corpora like LP and so on are not linked there). 

Also I wonder now if UMR is thought of as an AMRv2? Maybe then this can also be mentioned in the guidelines. Otherwise the links may be a little confusing since they kind of point out of the "AMR-universe" to another type of MR.
